### PR TITLE
fix(twitter): get current visiting identity via twitter's API

### PR DESCRIPTION
**Pull request by UncleBill**
it's faster

https://mask.atlassian.net/browse/MF-1133

### CODE REVIEW

File: packages/mask/src/social-network-adaptor/twitter.com/collecting/identity.ts [hunk 1]
```diff
@@ -116,14 +117,26 @@ function resolveLastRecognizedIdentityMobileInner(
     window.addEventListener('locationchange', onLocationChange, { signal: cancel })
 }
 
+const getFirstSlug = () => {
```
Comment (guanbinrui) [inline packages/mask/src/social-network-adaptor/twitter.com/collecting/identity.ts:115]: Why not compare the full URL?

Comment (UncleBill) [reply packages/mask/src/social-network-adaptor/twitter.com/collecting/identity.ts:115]: For example, https://twitter.com/VitalikButerin and https://twitter.com/VitalikButerin/with_replies both contain information of current visiting user

File: packages/mask/src/social-network-adaptor/twitter.com/collecting/identity.ts [hunk 2]
```diff
@@ -116,49 +112,75 @@ function resolveLastRecognizedIdentityMobileInner(
...
-    const avatarSelector = searchAvatarSelector()
-    const avatarMetaSelector = searchAvatarMetaSelector()
-    const assign = async () => {
-        await delay(5000)
-        const bio = getBio()
-        const nickname = getNickname()
-        const handle = getTwitterId()
-        const avatar = getAvatar()
-        const homepage = await Services.Helper.resolveTCOLink(getPersonalHomepage())
+    let firstSlug = getFirstSlug()
```
Comment (guanbinrui) [inline packages/mask/src/social-network-adaptor/twitter.com/collecting/identity.ts:154]: Does it still necessary after we have fetched the identity from the network?

**Comment by UncleBill** [comment]
resolved in https://github.com/DimensionDev/Maskbook/pull/6708

**Pull request closed without merging**